GitExtensionsVSIX/source.extension.vsixmanifest

```diff
@@ -13,7 +13,9 @@
     </Metadata>
 
     <Installation>
-        <InstallationTarget Version="[14.0,15.0)" Id="Microsoft.VisualStudio.Community" />
+        <InstallationTarget Version="[14.0,16.0)" Id="Microsoft.VisualStudio.Community" />
```

Comment:
I suppose cherry-pick. (I am not sure if it good fix, so I will not create PR)
Something like next:

```
git fetch https://github.com/azarkevich/gitextensions.git v2.49.01p:refs/heads/tmp
git cherry-pick 8b63e2f2d1e28c5b67fc88eb9cd8849b1b225250
git branch -D refs/heads/tmp
```